src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -38,22 +39,34 @@ import {
 import { Add16 as Add, TrashCan32 as Delete } from '@carbon/icons-react';
 
 import { ListPageLayout } from '..';
-import { sortRunsByStartTime } from '../../utils';
+import {
+  getDefaultCancelState,
+  setDefaultCancelState,
+  sortRunsByStartTime
+} from '../../utils';
 import {
   cancelPipelineRun,
   deletePipelineRun,
   rerunPipelineRun,
   startPipelineRun,
   useIsReadOnly,
   usePipelineRuns,
+  useProperties,
   useSelectedNamespace
 } from '../../api';
 
 export function PipelineRuns({ intl }) {
   const history = useHistory();
   const location = useLocation();
   const params = useParams();
+  const { data } = useProperties();
 
+  const { pipelinesVersion } = data;
+  const versionCheck = parseInt(pipelinesVersion.split('.')[1], 10);
+  const allowCancelOptions = !Number.isNaN(versionCheck) && versionCheck > 34;
```

Comment:
note to self: another option would be to use something like [`semver`](https://www.npmjs.com/package/semver) to perform the comparison, especially if we're going to end up with similar checks for other features. This could help with compatibility checks especially as Tekton Pipelines starts removing some of the v1alpha1 APIs (e.g. Conditions)

e.g. 
```js
const minVersion = '>=0.35.0';
const allowCancelOptions = semver.satisfies(pipelinesVersion, minVersion);
```

Depending on the size the package contributes to the build output we may want to import only the specific function used, there are some docs included for how to achieve that.